package.json

```diff
@@ -101,6 +101,16 @@
                 "default": {"app-boilerplate": "apk add gcc musl-dev python3-dev"},
                 "scope": "application",
                 "description": "Additional dependencies to install for each app."
+              },
+              "ledgerDevTools.keepContainerTerminal": {
+                "type": "boolean",
+                "default": true,
+                "markdownDescription": "Keep Terminal window opened when Container Update successfully."
```

Comment:
Reword : `Keep the Terminal window opened after a successful Container Update.`

Reply:
Done